examples/bluetooth/main.py

```diff
@@ -7,10 +7,10 @@
     adv = bt.get_adv()
     if adv 
         # try to get the complete name
-        print(bluetooth.resolve_adv_data(adv.data, Bluetooth.ADV_NAME_CMPL))
+        print(bt.resolve_adv_data(adv.data, Bluetooth.ADV_NAME_CMPL))
 
         # try to get the manufacturer data (Apple's iBeacon data is sent here)
-        print(binascii.hexlify(bluetooth.resolve_adv_data(adv.data, Bluetooth.ADV_MANUFACTURER_DATA)))
+        print(bt.hexlify(bluetooth.resolve_adv_data(adv.data, Bluetooth.ADV_MANUFACTURER_DATA)))
```

Comment:
This should be
```python
print(binascii.hexlify(bt.resolve_adv_data(adv.data, Bluetooth.ADV_MANUFACTURER_DATA)))
```

You've changed the wrong word

Reply:
Hi, @sdaniel55 thanks :smiley: